.github/workflows/test.yml

```diff
@@ -64,7 +64,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
-          python-version: '3.x'
+          python-version: '3.11.x'
```

Comment:
See https://github.com/eventlet/eventlet/issues/795
I reverted to `3.11.x` as ran in the previous build: https://github.com/ghostunnel/ghostunnel/actions/runs/6742314101/job/18328220881?pr=407

If 3.12+ is wanted in the future, please fix the test using the deprecated / removed method in a separate PR as this was not related to any of my changes.